### ISSUE TITLE
Inform translators when GatherPress is available on the plugin repo

### ISSUE THREAD

**Issue by patriciabt** [GatherPress/gatherpress#542]
Once the plugin is in the repository, reach out to current translators as well as potential translators of many other languages with the link to the translate.wordpress.org GatherPress entry.
Existing languages: not sure if it adds automatically once the plugin is there, or each translator can upload the existing .po file for each to be credited for their translations

**Comment by patriciabt** [comment]
once there, also empty /languages (all files but gatherpress.pot)

**Comment by javiercasares** [comment]
WordPress allows uploading PO files (the translated ones) to GlotPress… so, when the languages are published on translate.wordpress.org we can upload the files we have to the platform.

Josep and I do that frequently because usually is simple to work on your computer and then upload the files than translate string by string on the platform (when there are hundreds or thousands of strings).

**Comment by javiercasares** [comment]
No. GlotPress doesn't use the PO files (nor the POT). This way of using translation is only for plugins not in the repo.

When the plugin is in the repo, each one of us should upload their translations if we want the credit for those.

**Comment by patriciabt** [comment]
Thank you, that what I thought!

**Comment by jeffpaul** [comment]
TLDR: once GatherPress is released on WP.org, in a subsequent release we should be able to remove the `/languages` directory as those contents are ingested into translate.wordpress.org and served up as language packs from there

See: https://make.wordpress.org/meta/handbook/tutorials-guides/translations/

Relevant bits:
> The WordPress.org plugin directory has [the translations import mechanism currently enabled for all plugins](https://make.wordpress.org/plugins/2015/12/06/plugin-translations-for-all-plugins/).

> For a plugin to be imported properly and take advantage of language packs, it must have a text domain that matches its slug (e.g., a plugin with the “akismet” slug must have a text domain of “akismet”) and must not have more than one text domain.

> Additionally, if the plugin already ships language files (.mo), these will be imported into translate.wordpress.org.

> Once a plugin has been imported and the relevant language packs have been generated, plugins can remove the language files from their package (and from SVN).

**Comment by patriciabt** [comment]
@jeffpaul yes thanks, the language folder is there only until the plugin is on the official repository

**Comment by patriciabt** [comment]
All past translators have been informed or are aware the plugin reached the repository and is now available to translate on 
https://translate.wordpress.org/projects/wp-plugins/gatherpress/
